Sources/Common/Parallel.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import NIO
+
+func paddedId(_ id: Int32) -> String {
```

Comment:
this can also be achieved with `String(format: "%03d", id)` I believe

Reply:
nice, updated